odl/trafos/wavelet.py

```diff
@@ -301,7 +323,8 @@ def adjoint(self):
             if `is_orthogonal` is ``False``
         """
         if self.is_orthogonal:
-            return self.inverse
+            scale = 1 / self.domain.partition.cell_volume
```

Comment:
Another option for this is to move this into the `_call` method so that the method actually becomes orthogonal, I guess that is preferable?